New package: SGCRNAs v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: SGCRNAs
- Repository: https://github.com/C37H41N2O6/SGCRNAs
- Created by: @C37H41N2O6
- Version: v1.0.0
- Commit: 58b5aaeba6c4116bc214dfa8dfb5d99f546bf0ca
- Git reference: HEAD

goerz commented:
Could you make sure that the README includes a brief description of what the package does, to have some context? Also, I noticed that the links to the documentation currently do not work.

Without context, it’s hard to give more specific feedback, but the [package naming guidelines](https://pkgdocs.julialang.org/v1/creating-packages/#Package-naming-rules) generally frown upon “random-letter” acronyms as package names.

C37H41N2O6 commented:
@goerz 
Thank you for your comment. I'm currently making revisions, including the points you mentioned.

goerz commented:
You may also want to consider what the appropriate initial version number should be. If you are following [semantic versioning](https://semver.org), a version number of 1.0 would indicate that the API is "stable" and that you will track breaking and non-breaking changes according to what is documented to be the public API of the package. That generally requires full documentation and, ideally, reasonable test coverage, in order to really be able to track those breaking vs non-breaking changes. There's nothing to say that an initial registration shouldn't be at v1.0, if you feel that it is ready for to be labeled "stable". However, if you are still tweaking things, it maybe be better to start with, e.g., v0.1 and then switch to `v1.0` when you feel the package is in good shape. There's no problem with moving from `v0.x` to `v1.0`, just to mark the API stable.

C37H41N2O6 commented:
@goerz 
Thank you for your suggestion regarding the version number. Except for addressing errors that I didn’t notice during verification or responding to feature requests from users, this is intended to be the final version, so I’m planning to keep it as v1.0.0.

I’ve also addressed the comments you provided a few days ago.
As I’m not very familiar with systems like Git, I would greatly appreciate it if you could advise me on what steps I should take next.

goerz commented:
You have to fix

> Repo URL does not end with /name.jl.git, where name is the package name

by [renaming the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository). After that, create a new registration (the same way you created this one).

> As I’m not very familiar with systems like Git

That’s probably not sustainable in the long run 😉. I recommend https://git-scm.com/book/en/v2

C37H41N2O6 commented:
Thank you very much for your guidance.
I will resend it via JuliaHub. Also, thank you for recommending the reference book.
Regarding the repository name, it is correctly set as `SGCRNAs.git`, not `SGCRNAs.jl.git`.
I also searched the contents of the files, and apart from the `sitename` and `canonical` in `docs/make.jl`, I couldn't find any instance of the latter string.
Although I don’t recall making any changes, I’ll try submitting it again in its current state.

JuliaHubRegistrator committed:
New package: SGCRNAs v1.0.0

UUID: 4c411929-775e-4429-ac3f-feb95d997042
Repo: https://github.com/C37H41N2O6/SGCRNAs.git
Tree: ebe4a169f71c92223d5746aa6c8204d081b870b4

Registrator tree SHA: d5716b7a540e5fbc43640c2fff2906fe50e9525a

goerz commented:
> Regarding the repository name, it is correctly set as SGCRNAs.git, not SGCRNAs.jl.git.

No: it is a _requirement_ for Julia packages that the repository name includes the `.jl` suffix. You have to rename the repository on GitHub and add `.jl`
